client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -164,20 +159,40 @@
     @Action hideAllVariables;
     @Action showAllVariables;
 
+    calcPolygon (variable: HMI.SimulationVariable): void {
+      const numRuns = variable.values.length;
+      const xArr = new Array(numRuns - 1);
+      const yMinArr = new Array(numRuns - 1);
+      const yMaxArr = new Array(numRuns - 1);
+
+      for (let runIndex = 0; runIndex < numRuns - 1; runIndex++) {
+        const run = variable.values.shift();
+        run.forEach((point, pointIndex) => {
+          xArr[pointIndex] = point.x;
+          yMinArr[pointIndex] = Math.min(yMinArr[pointIndex] ?? point.y, point.y);
+          yMaxArr[pointIndex] = Math.max(yMaxArr[pointIndex] ?? point.y, point.y);
+        });
+      }
+
+      variable.polygon = [
+        ...yMinArr.map((y, i) => ({ x: xArr[i], y })),
+        ...yMaxArr.map((y, i) => ({ x: xArr[i], y })).reverse(),
+      ];
+    }
+
     get variables (): HMI.SimulationVariable[] {
       let variables = this.getSimModel(this.modelId).variables;
       variables = _.cloneDeep(variables);
       variables.map(variable => {
         variable.styles = variable.styles || [];
-        for (let i = 0; i < variable.values.length; i++) {
-          if (variable.values.length > 5) {
-            i !== variable.values.length - 1
-              ? variable.styles.push(mergeStyle(OTHER_STYLE, NO_LINE_STYLE))
-              : variable.styles.push(mergeStyle());
-          } else {
-            variable.styles.push(mergeStyle(i !== variable.values.length - 1 && OTHER_STYLE));
+        if (variable.values.length > 5) {
```

Comment:
I think we should have this arbitrary number of runs as a constant in this panel. Something like `THRESHOLD = 5`